eks_fargate/README.md

```diff
@@ -144,6 +152,47 @@ spec:
 
 **Note**: Add your desired `kube_cluster_name:<CLUSTER_NAME>` to the list of `DD_TAGS` to ensure your metrics are tagged by your desired cluster. You can append additional tags here as space separated `<KEY>:<VALUE>` tags.
 
+#### Running the Cluster Agent / Cluster Checks Runner
+
+Running the Cluster Agent is recommended as it is required for a number of features: events collection, Kubernetes resources view, cluster checks, etc.
```

Comment:
would be nice to link the corresponding docs for `events collection`, `Kubernetes resources view`, `cluster checks`
https://docs.datadoghq.com/agent/kubernetes/?tab=helm#event-collection
https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-view 
https://docs.datadoghq.com/agent/cluster_agent/clusterchecks/#overview

Reply:
Can you add add the reference links to the bottom of the doc?

[29]: https://docs.datadoghq.com/agent/kubernetes/?tab=helm#event-collection
[30]: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-view
[31]: https://docs.datadoghq.com/agent/cluster_agent/clusterchecks/#overview

```suggestion
Datadog recommends you run the Cluster Agent to access features such as events collection, Kubernetes resources view, and cluster checks.
```